### DISCUSSION TITLE
Welcome to openSn Discussions!

### DISCUSSION THREAD

**Discussion by ragusa** [Open-Sn/opensn#37] [Announcements]
## 👋 Welcome!
🆘 If you need help, [use "Ask for Help"](https://github.com/Open-Sn/opensn/discussions/categories/ask-for-help) 

  Here, we’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  
To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yingchiyu** [comment]
Hello, I'm following the steps in install_linux.md and install_linux_easy.md on wsl1 (Ununtu 20.04) and I'm having a problem:
After executing the “./configure.sh” command in the opensn directory, I followed the prompt to execute the “set_opensn_env” file to set the environment variables. 
However, I get the following error when I execute the “make” command in the /opensn/build directory:

/usr/bin/ld: ../libopensn.so: undefined reference to `SNESGetConvergedReasonString'
/usr/bin/ld: ../libopensn.so: undefined reference to `KSPGMRESSetBreakdownTolerance'
collect2: error: ld returned 1 exit status
make[2]: *** [lua/CMakeFiles/opensn.dir/build.make:101: lua/opensn] Error 1
make[1]: *** [CMakeFiles/Makefile2:200: lua/CMakeFiles/opensn.dir/all] Error 2
make: *** [Makefile:91: all] Error 2

I have tried various ways to solve it but failed, I don't know what is causing it, I would appreciate if you could provide me with help!
![error](https://github.com/Open-Sn/opensn/assets/116443237/c0fa3537-318d-4faf-add4-3f7fc4129d7c)

**Reply by andrsd** [reply]
Hi! If you are looking for user support, please, open a ~~[new discussion in the Q&A category](https://github.com/Open-Sn/opensn/discussions/new?category=q-a)~~ [new discussion the Ask for Help category](https://github.com/Open-Sn/opensn/discussions/new?category=ask-for-help). Thanks!
